src/clients/UBAClient/UBAClientBase.ts

```diff
@@ -194,15 +194,14 @@ export abstract class BaseUBAClient extends BaseAbstractClient {
   protected computeLpFee(
     amount: BigNumber,
     depositChainId: number,
-    hubPoolChainId: number,
```

Comment:
I can see that `computeLpFee()` is called by `computeSystemFee()`, except in that case the chainId might _not_ be `hubPoolChainId`. The caller may instead pass in any destination chain ID. If so, is it correct to name this `hubPoolChainId`?